src/models/constructor-items/sub-blocks.ts

```diff
@@ -143,7 +143,8 @@ export interface BannerCardProps {
     disableCompress?: boolean;
     color?: ThemeSupporting<string>;
     theme?: TextTheme;
-    button: Pick<ButtonProps, 'text' | 'url' | 'target'>;
+    button: Pick<ButtonProps, 'text' | 'url' | 'target' | 'theme'>;
+    mediaView?: 'fit' | 'full';
```

Comment:
but in header we are using the same format. I want to keep consistence view

Reply:
if we wanna get it same in Header and Banner, suppose we should create type, `type MediaView = 'fit' | 'full'` to use for both and keep same values.
In case we want to make them different (use `contain` in Banner) lets name theme different

Reply:
added MediaView type